GettingStarted.tex

```diff
@@ -13,16 +23,43 @@
 \tlcTitlePageAndTableOfContents
   {Getting Started}
   {Gary Allan Howard}
-  {Getting Started guide covers how to install \tlca\ both globally and
-    locally, describes the general use case, how to customize your \tlca\
-    environment, describes the commands \tlca\ implements, and reveals the
-    packages \tlca\ depends upon.}
+  {Getting Started guide covers how to install \tlcA\ both globally and
+    locally, describes the general use case, how to customize your \tlcA\
+    environment, describes the commands \tlcA\ implements, and reveals the
+    packages \tlcA\ depends upon.}
 
 % ------------------------------------------------------------------------------
 
 \section{Installation}
+This section describes how to install \tlcA\ either globally to make it avalilbe
+to your \LaTeX\ environment or locally to the document you are authoring.
+
 \subsection{Global installation}
+A global installation is done by installing \tlcA\ into your /path/to/your/texmf
+directory.  Assuming a TexLive installation exists at \texDist\ the following
+shell commands will make \tlcA\ available to your \LaTeX\ environment.
+
+\begin{enumerate}
+  \item cd \tlcHome
+  \item git clone \tlcRepo\
+  \item cd \tlcProduct\
+  \item mkdir \tlcGlobalDist
```

Comment:
On linux, you must pass the -p parameter to create a nested subdirectory.

e.g. sudo mkdir -p $(kpsewhich -var-value TEXMFLOCAL)/tex/latex/tlc-article